main.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'app'
+
+def list_options
+  '
+      1 - Add a book
+      2 - Show all books
+      3 - Add a person
+      4 - Show all people
+      5 - Add a rental
+      6 - Show all rentals for a given ID
+      7 - Exit
+    '
```

Comment:
- [ ] Good job with the prompt arrangement, but please kindly rearrange it so that it matches the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)